### PR TITLE
add habitlist

### LINKED ISSUE #2: User Story for habit list display
## Value Proposition
As a user, I want to have a view of my habit, so I can keep a track of it.

## Description

Screenshot 2022-10-17 at 16 13 37

## Acception Criteria
- [x] as a user I want to have list of All Day doings displayed 
- [x] as a user I want to list them under AllDay

## Tasks
- [x] new branch
- [x] create a Headline `All Day`
- [x] create a list for daily habits

### PR DESCRIPTION
**Pull request by Sukhpreet92**

### CODE REVIEW

File: components/HabitList.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import { HABITS } from "../lib/db";
+
+function Card({ habit }) {
+  return (<ul>{habit.id}</ul>), (<p>{habit.name}</p>);
```
Comment (F-Kirchhoff) [inline components/HabitList.js]: this doesn't seem correct, a ul with only text in it, and you try to return 2 elements inside the card?

File: components/HabitList.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export function HabitList() {
+  return (
+    <Main>
+      <h3>All Day</h3>
+      <ul>
+        {HABITS.map((habitElement) => {
+          return <Card key={habitElement.id} habit={habitElement} />;
+        })}
+      </ul>
+    </Main>
```
Comment (F-Kirchhoff) [inline components/HabitList.js]: I guess you should refactor this part, the HabitList should only be the ul list, Main and h3 can be put into the index.js

File: lib/db [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    name: "shower",
+    categorie: "all-day",
+  },
+  {
+    id: 3,
+    name: "drink water",
+    categorie: "all-day",
+  },
+  {
+    id: 4,
```
Comment (F-Kirchhoff) [inline lib/db]: You forgot the '.js' ending on the file

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
This looks like a good start! However, there are some issues with your react code, please make sure to have only one component per file, so export your Card into a new file, also make sure that the return statement of your component is only one single react element (with nested child elements if needed)

File: components/HabitList.js (hunk 2 above)
Comment (Sukhpreet92) [reply components/HabitList.js:16]: just to make sure. the HabitList is not required I can move  

   ``` <h3>All Day</h3>
       <ul>
         {HABITS.map((habitElement) => {
           return <Card key={habitElement.id} habit={habitElement} />;
         })}
       </ul> 
```


to index.js

File: components/HabitList.js (hunk 1 above)
Comment (Sukhpreet92) [reply components/HabitList.js:5]: is it fine if I export the function as Habit to index.js with single element?

File: components/HabitList.js (hunk 2 above)
Comment (Sukhpreet92) [reply components/HabitList.js]: > I guess you should refactor this part, the HabitList should only be the ul list, Main and h3 can be put into the index.js


just to make sure. the HabitList is not required I can move  

   ``` <h3>All Day</h3>
       <ul>
         {HABITS.map((habitElement) => {
           return <Card key={habitElement.id} habit={habitElement} />;
         })}
       </ul> 
```


to index.js

File: components/HabitList.js (hunk 1 above)
Comment (Sukhpreet92) [reply components/HabitList.js]: > this doesn't seem correct, a ul with only text in it, and you try to return 2 elements inside the card?

is it fine if I export the function as Habit to index.js with single element?

**Review by anhthu253 — APPROVED** [review]
alle ok

**Pull request merged**